commons/zenoh-util/src/std_only/net/mod.rs

```diff
@@ -425,12 +437,12 @@ pub fn get_ipv4_ipaddrs() -> Vec<IpAddr> {
         .collect()
 }
 
-pub fn get_ipv6_ipaddrs() -> Vec<IpAddr> {
+pub fn get_ipv6_ipaddrs(interface: Option<String>) -> Vec<IpAddr> {
```

Comment:
It seems this function does not need to take ownership of the `String`, maybe pass `Option<&str>`?